### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by lordforall**
Adding CNFT pool

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -15,6 +15,22 @@
...
+    "social": {
+      "website": "CNFTpool.com",
+      "discord": "",
+      "reddit": "",
+      "instagram": "",
+      "facebook": "",
+      "youtube": "https://www.youtube.com/channel/UCpz7w-c5CNqEyp6XW1YMkcA",
+      "twitter": "LordForAll"
+    },
+    "telegram": "",
```
Comment (Bez625) [inline registry.json:30]: The alliance rules are that you must supply the TG handle of an individual (not group or channel)

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (lordforall) [reply registry.json]: Has been updated now thanks

**Review by lordforall — COMMENTED** [review]
fixed telegram

**Comment by lordforall** [comment]
all fixed

**Pull request merged**